### ISSUE TITLE
deck submission, select format

### ISSUE THREAD

**Issue by Razor22Razor** [Slanfan/MTG-Tolaria#320]
When submitting a deck, it would be very nice to be able to select from a drop-down menu which format it is for. 

Good for database sorting, too

**Comment by Slanfan** [comment]
Archetypes are a hassle and too many opinions around it. To have a strict way of doing this it would need a rally structured way to determine what's what. 

With that said. Deck legality checker is something that I have had in mind all along and now we have an improvement for it to follow. 😎

**Comment by LoekieMtg** [comment]
True, I didnt specificy what I meant. I meant the base archetypes, Aggro, Aggro-Combo, Aggro-Control, Control, Combo. See this article from Rich 
https://docs.google.com/document/d/1-HMyZWVxlJ9n8LV91UZRGpg1shQcIz3z_Rqq0WcL1kU/edit?fbclid=IwAR0wuOTIzRyiX6xAgbRA36_939WFaHeE94sIu8Lslb9JZTta2DPQ4YfH3Cg
